src/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenCommonParameters.cs

```diff
@@ -35,6 +36,6 @@ internal class AcquireTokenCommonParameters
         public ApiEvent.ApiIds ApiId { get; set; } = ApiEvent.ApiIds.None;
         public IEnumerable<string> Scopes { get; set; }
         public Dictionary<string, string> ExtraQueryParameters { get; set; }
-        public string AuthorityOverride { get; set; }
+        public AuthorityInfo AuthorityOverride { get; set; }
```

Comment:
>AuthorityOverride [](start = 29, length = 17)

should this simply be AuthorityInfo?

Reply:
It can be, but it was in the original API as the AuthorityOverride since you're overriding the default application's authority value.  open to ideas here, but this is internal so it's easy to change if we want.

---
In reply to: [250003542](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/815#discussion_r250003542) [](ancestors = 250003542)